lib/utils.js

```diff
@@ -251,9 +251,43 @@ function getPackageVersion (pkgName) {
   return pkgInfo.version;
 }
 
+/**
+ * Pulls the initial values of Appium settings from the given capabilities argument.
```

Comment:
Is it helpful to add https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/settings.md around here?

Reply:
will add